packages/contract-wrappers/src/types.ts

```diff
@@ -225,3 +225,61 @@ export interface DutchAuctionData {
     beginTimeSeconds: BigNumber;
     beginAmount: BigNumber;
 }
+
+export interface CoordinatorServerApprovalResponse {
+    signatures: string[];
+    expirationTimeSeconds: BigNumber[];
+}
+export interface CoordinatorServerApprovalRawResponse {
+    signatures: string[];
+    expirationTimeSeconds: BigNumber;
+}
+
+export interface CoordinatorServerCancellationResponse {
+    outstandingFillSignatures: [
+        {
+            orderHash: string;
+            approvalSignatures: string[];
+            expirationTimeSeconds: BigNumber;
+            takerAssetFillAmount: BigNumber;
+        }
+    ];
+    cancellationSignatures: string[];
+}
+
+export interface CoordinatorServerResponse {
+    isError: boolean;
+    status: number;
+    body?: CoordinatorServerCancellationResponse | CoordinatorServerApprovalRawResponse;
+    error?: any;
```

Comment:
Why `any`?

Reply:
Hmm. on second thought, this could be `ValidationError[]` with the type imported from the server. I guess the server will never return any json that isn't a success response known error type.

Reply:
Looking closer at the server response types, looks like the error response isn't exactly the formally defined type either. 

Response: https://github.com/0xProject/0x-coordinator-server/blob/master/ts/src/middleware/error_handling.ts#L34-L47

Exported type: https://github.com/0xProject/0x-coordinator-server/blob/master/ts/src/errors.ts#L19

Leaning towards leaving it as `any`, at least for now. This error field is never surfaced on its own, it's just for additional details from the server. The consumer always has access to other informative fields.

The data in this field will either be undefined or some kind of validation error so would have to be surfaced to end user anyway right? As in, even if we strongly type this field, it's not likely that the developer would be able to do automated error correction?